app/validators/enforce_sso_validator.rb

```diff
@@ -2,7 +2,7 @@
 
 class EnforceSSOValidator < ApplicationValidator
 
-  SSO_NOT_TESTED_ERROR = 'SSO could not be enforced. Authentication flow has to be checked.'
+  SSO_NOT_TESTED_ERROR = 'SSO could not be enabled. Authentication flow has to be checked.'
```

Comment:
It seems to me `enforced` is more correct. Because at this point SSO will be enabled and can be used to login along with password authentication, but it's not the only accepted authentication mechanism. The `SSO Enforce` switch disables password authentication.